File: 01.Linear Search/linear_search.java
public class linear_search {
    public static void main(String args[]){

        int NumberList[] = {10,20,30,40,50,60,70,80,90,100};
        int find = 20;

        if(linear_search(NumberList, NumberList.length, find)){
            System.out.println("Found");
        }
        else{
            System.out.println("Not found");
        }

    }


    public static boolean linear_search(int List[], int Size, int var){

        boolean result = false;
        for(int i = 0; i < Size; i++){
            if(List[i] == var){
                System.out.println(var + " at " + i + " Matched");
                result = true;
            }
            else {
                System.out.println(var + " at " + i + " Not matched");
            }
        }
        return result;

    }
}

File: 01.Linear Search/linear_search.c
#include <stdio.h>

int linear_search(int array[], int size, int var);

int main()
{
    int numbers[] = {10,20,30,40,50,60,70,80,90,100};
    int find = 20;

    if(linear_search(numbers, sizeof(numbers)/4, find)){
        printf("Found\n");
    }
    else{
        printf("Not found\n");
    }

    return 0;
}


int linear_search(int array[], int size, int var){

    int i, found = 0;

    for (i = 0; i < size; i++){
        if(array[i] == var){
            printf("%d at %d Matched\n", var, i);
            found = 1;
        }
        else{
            printf("%d at %d Not matched\n", var, i);
        }
    }
    return found;
}
